src/data/h1/tags/weapon.yml

```diff
@@ -454,34 +484,48 @@ comments:
               en: ...
         - name: error
           en: >
-            The function of this field is unknown. It is **possibly unused by the game**.
+            The function of this field is unknown. It is **unused by the game**.
             Although labeled in [Guerilla][] as a range \[0, 1\], the pistol has a max
             value of 2. Setting this to 0 does not affect projectile spread.
         - name: error acceleration time
           en: >
             Affects how long it takes for the weapon to reach its maximum projectile
-            error angle during rapid or automatic fire.
+            error angle while holding the trigger. Can be negated using quick
+            trigger taps.
         - name: error deceleration time
           en: >
             Affects how long it takes for the weapon to reach its minimum projectile
             error angle during a pause in firing.
         - name: charging time
-          en: ...
+          en: >
+            The amount of time the user has to hold the trigger before the weapon
+            is considered charged.
         - name: charged time
-          en: ...
+          en: >
+            The amount of time it takes _after_ the weapon becomes charged before
+            the overcharged behavior takes effect. **Unused if overcharged action
```

Comment:
I've taken to italicizing/emphasizing field names whenever called out, so `_overcharged action_`.